### ISSUE TITLE
Idea: Add a way to report story issues from within the story

### ISSUE THREAD

**Issue by ptoro** [rgerum/unofficial-duolingo-stories#124]
Proposed by バリウム on Discord

Motivation: not all users will have the desire or ability to join the Discord server to report story issues.

We could add a bug icon (floating on the bottom right of the screen) like Duolingo's:
![image](https://user-images.githubusercontent.com/4418961/177370772-273de7d2-e184-4432-b032-c47020f50478.png)

Clicking it would open a form in a popup on the page to report an issue:
<img width="919" alt="image" src="https://user-images.githubusercontent.com/4418961/177370887-dd5bb1d5-ebe1-47ca-a41a-e9e3d1c4be77.png">

That report could go to a Discord channel where story contributors could review it and fix it if necessary.

**Comment by jahtomini** [comment]
I would like to work on this if the feature is approved. What do you think @rgerum ?

**Comment by rgerum** [comment]
Yes sounds good. I think if we want to pipe them to discord the first thing to do would be to have a look at the discord api how to sent messages to a discord channel.

**Comment by jahtomini** [comment]
> Yes sounds good. I think if we want to pipe them to discord the first thing to do would be to have a look at the discord api how to sent messages to a discord channel.

Exactly, I'll take a look at the Discord API docs. I currently have a project commitment but once I'm done, I'll get to work on it. Can I be assigned this task?

**Comment by ptoro** [comment]
> Can I be assigned this task?

Certainly! Let an admin know when you're ready, and one of us will assign it to you.
